microsoft-edge/webview2/release-notes.md

```diff
@@ -10,19 +10,29 @@ ms.date: 04/13/2023
 ---
 # Release Notes for the WebView2 SDK
 
-The WebView2 team updates the [WebView2 SDK](https://www.nuget.org/packages/Microsoft.Web.WebView2) on a four-week cadence. This article contains the latest information on product announcements, additions, modifications, and breaking changes to the APIs.
+The WebView2 team updates the [WebView2 SDK](https://www.nuget.org/packages/Microsoft.Web.WebView2) on a four-week cadence.  This article contains the latest information on product announcements, additions, modifications, and breaking changes to the APIs.
 
 Generally, release notes apply across the supported platforms, which are listed in [WebView2 API Reference](webview2-api-reference.md).
 
-WebView2 changes may require an update to the Runtime, SDK, or both. Most new APIs require both Runtime and SDK updates. Starting with the February 2023 release, the update requirement for each bug fix is indicated as follows:
+WebView2 changes may require an update to the Runtime, SDK, or both.  Most new APIs require both Runtime and SDK updates. Starting with the February 2023 release, the update requirement for each bug fix is indicated as follows:
 
 | Indicator | Meaning |
 |---|---|
 | No label | Both the Runtime and the SDK need to be updated. |
 | **Runtime-only** | Only the Runtime needs to be updated. |
 | **SDK-only** | Only the SDK needs to be updated. |
 
-WebView2 shares code and binaries with the Microsoft Edge browser, and is released around the same time. As a result, WebView2 Runtime releases generally also include Microsoft Edge updates. For Microsoft Edge updates, see [Release notes for Microsoft Edge Stable Channel](/deployedge/microsoft-edge-relnote-stable-channel) and [Release notes for Microsoft Edge Beta Channel](/deployedge/microsoft-edge-relnote-beta-channel).
+WebView2 shares code and binaries with the Microsoft Edge browser, and is released around the same time.  As a result, WebView2 Runtime releases generally also include Microsoft Edge updates.  For Microsoft Edge updates, see [Release notes for Microsoft Edge Stable Channel](/deployedge/microsoft-edge-relnote-stable-channel) and [Release notes for Microsoft Edge Beta Channel](/deployedge/microsoft-edge-relnote-beta-channel).
+
+
+<!-- ------------------------------ -->

```

Comment:
tech review this section
@novac42 @peiche-jessica @nishitha-burman 

In every Promotions section, wrote either:
Prerelease > Promotions:
The following APIs have been promoted from experimental to prerelease stable.

Release > Promotions:
The following APIs have been promoted from prerelease stable to release.

Is that technically correct in all or most cases?

Reply:
Can we use something like 'Prerelease-stable" instead of "Stable"? I understand the 3 init-cap phases myself but not sure if developers would think "isn't Stable and Release the same thing?"

Reply:
@novac42 
I'm going to push a commit that doesn't add new terminology / nomenclature such as "Prerelease-stable" or "production release", but just clarifies & tightens use of terms "Experimental, Stable, Release".

The latest wording in "Promotions" sections:
The following APIs have been promoted from Experimental to Stable in this Prerelease SDK.
The following APIs have been promoted to Release.

Reply:
@novac42 - status EOD Monday: re:
"Can we use something like 'Prerelease-stable" instead of "Stable"?"
My latest wording doesn't attempt that pattern.  I tried to do a clean, consistent implementation of existing terminology usage per Jessica, rather than creating any new terminology/nomenclature or status names.

Pls evaluate latest wording, maybe in later PR we can adopt & define a new status name.  Or in this PR, which already is an improvement.